sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -1145,6 +1136,11 @@ public abstract partial class ObjectSerializer
         public abstract System.Threading.Tasks.ValueTask SerializeAsync(System.IO.Stream stream, object? value, System.Type inputType, System.Threading.CancellationToken cancellationToken);
         public virtual System.Threading.Tasks.ValueTask<System.BinaryData> SerializeAsync(object? value, System.Type? inputType = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
+    public partial class RawContentOptions
+    {
+        public RawContentOptions() { }
+        public bool UseCamelCaseNamingConvention { get { throw null; } set { } }
```

Comment:
I wanted to show this way of naming it to encourage discussion.  I think it's probably better (to holistically include CosmosSerializationOptions and improve extensibility for later conversions) if we make what we're now calling PropertyNamingConvention a public enum and use those values to set this property.

Reply:
To give credit where due, this [name comes from EntityFramework](https://github.com/efcore/EFCore.NamingConventions#supported-naming-conventions).